### PR TITLE
docs: fix english errors in readme

### PR DESCRIPTION
**Pull request by dnalien22**
**Description of changes**
 Fixed English errors in the document.

**GitHub issues resolved by this PR**
 N/A

**Quality Assurance**
Once the changes in this PR are merged and deployed, success criteria is:
Add clear instructions for the users with more accurate English.

### CODE REVIEW

**Comment by dnalien22** [comment]
Tried my best here.

File: README.md [hunk 1]
```diff
@@ -1,45 +1,49 @@
-[![MIT License](https://img.shields.io/badge/license-MIT_License-green.svg?style=flat-square)](https://github.com/devopness/devopness/blob/main/LICENSE)
+git checkout -b <onboarding_daniel>
```
Comment (Diegiwg) [inline README.md:1]:
```suggestion
```

This information should not be here.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
 # Devopness - Official open-source repository
 
 ## About Devopness
-`Devopness` aims to drastically simplify the way we manage cloud applications and multi cloud infrastructure in a secure and productive way.
+`Devopness` aims to drastically simplify the way we manage cloud applications and multi-cloud infrastructure in a secure and productive way.
 
-By streamlining essential DevOps practices we're making first class software deployment and cloud infrastructure management tools accessible and affordable to everyone in the world who is involved in software projects.
+By streamlining essential DevOps practices, we're making first-class software deployment and cloud infrastructure management tools accessible and affordable to everyone in the world who is involved in software projects.
 
-## 📚 <a id="docs"></a>Documentation
```
Comment (Diegiwg) [inline README.md:11]: What is the explanation for removing the 'a' tag?

**Review by Diegiwg — CHANGES_REQUESTED** [review]

**Comment by Diegiwg** [comment]
Use the PGs below as a reference to create the Description and Title of your PR, please:

- https://github.com/devopness/devopness/pull/894
- https://github.com/devopness/devopness/pull/857
- https://github.com/devopness/devopness/pull/829

File: README.md (hunk 2 above)
Comment (dnalien22) [reply README.md:11]: I returned the 'a' back to it's original place.

File: README.md (hunk 1 above)
Comment (dnalien22) [reply README.md]: I removed it.

**Review by Diegiwg — APPROVED** [review]

**Comment by dnalien22** [comment]
I'm having trouble trying to update the last commit @Diegiwg, is it lack of permission?

**Comment by Diegiwg** [comment]
> I'm having trouble trying to update the latest commit @Diegiwg, is it lack of permission? Changes in this commit: _multi-cloud_ > multi cloud _end-user_ > end user

This is what fixed the issue for me:

Click on Edit in the top right corner.
Click on the base selector and choose the same base branch.
GitHub will ask you whether you want to change base, confirm.
The PR should update and the commit should show up.

Reference: https://stackoverflow.com/questions/45626986/updating-pull-request-after-review-changes-commit-doesnt-show-up-in-the-pull-re

**Comment by Diegiwg** [comment]
@dnalien22, in regarding your commit messages, always try to use the following pattern:

- https://www.conventionalcommits.org/en/v1.0.0/#summary
- https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines

File: README.md [hunk 3]
```diff
@@ -1,45 +1,46 @@
-[![MIT License](https://img.shields.io/badge/license-MIT_License-green.svg?style=flat-square)](https://github.com/devopness/devopness/blob/main/LICENSE)
+[![MIT License](https://img.shields.io/badge/license-MIT-green.svg?style=flat-square)](https://github.com/devopness/devopness/blob/main/LICENSE)
```
Comment (Diegiwg) [inline README.md:1]: Well noted. The `License` in the badge was redundant.

**Review by Diegiwg — APPROVED** [review]

**Comment by dnalien22** [comment]
> > I'm having trouble trying to update the latest commit @Diegiwg, is it lack of permission? Changes in this commit: _multi-cloud_ > multi cloud _end-user_ > end user
> 
> It's not a 'permission' issue.
> 
> Sometimes, synchronization problems happen, so you can try to resolve them by doing a manual synchronization.
> 
> This is what fixed the issue for me:
> 
> 1. Click on Edit in the top right corner.
> 2. Click on the base selector and choose the same base branch.
> 3. GitHub will ask you whether you want to change base, confirm.
> 4. The PR should update and the commit should show up.
> 
> Reference: https://stackoverflow.com/questions/45626986/updating-pull-request-after-review-changes-commit-doesnt-show-up-in-the-pull-re

Thank you so much It worked!

**Review by thlmenezes — APPROVED** [review]

**Pull request merged**